numpy/ma/tests/test_core.py

```diff
@@ -993,6 +993,18 @@ def test_object_with_array(self):
         mx[1].data[0] = 0.
         assert_(mx2[0] == 0.)
 
+    def test_saving_exception(self):
+
+        # make sure save and savez return exception for masked arrays
+        mx1 = masked_array([1.], mask=[True])
+        exc_text = 'fromfile() not yet implemented for a MaskedArray'
```

Comment:
```suggestion
        exc_text = 'MaskedArray.tofile() not implemented yet.'
```

Reply:
Pytest now works locally after recreating the conda env.  
exc_text should actually be a regex expression, so with r'MaskedArray\.tofile\(\) not implemented yet.', the test passes.

Reply:
github removed the backslash before the dot and each parenthesis... I'll push the code tomorrow.